### DISCUSSION TITLE
False positive

### DISCUSSION THREAD

**Discussion by azurenekowo** [infinitered/nsfwjs#703] [Model Training]
![unknown (4)](https://user-images.githubusercontent.com/67498765/156519707-a62653d6-5b7b-4ffa-a139-134538c0f8b9.png)
 
I don't know how to describe this image in the bug report title, but _somehow_ it managed to get flagged as "Hentai" for **90%** on https://nsfwjs.com and **92.7%** in the Node.JS example!

The 90% 2.6MB mobilenet module:
![image](https://user-images.githubusercontent.com/67498765/156519808-3e43c51e-5f88-42f6-97ab-39cb6a1f3e6d.png)

![image](https://user-images.githubusercontent.com/67498765/156520507-ede1365e-8571-4524-9a2e-d00c5c6f3ca3.png)

**Comment by GantMan** [comment]
Thanks for reporting this.  I'll add this image and images like this to the training set.

**Comment by azurenekowo** [comment]
@GantMan I'm glad to hear about that! and also, can I ask why is there a lot of false positive on mangas / black-colored doujins?

**Comment by GantMan** [comment]
That's a good question.  My guess is that manga wasn't properly represented in the training set.  However, the model is set to have more false positives, than false negatives.  Because it's better to error on caution.

**Comment by azurenekowo** [comment]
Sure, thanks for explaining sir!
